components/Builder.tsx

```diff
@@ -169,25 +176,42 @@ export default function Builder({ data }: { data: any }) {
         />
       </div>
 
-      <div className="flex gap-2 justify-center items-center">
-        {isSignedIn && (
-          <SaveButton
-            formation={formation}
-            artifact={artifact}
-            layout={layout}
-            user={user}
+      <div className="flex gap-2 items-center justify-between sm:w-full w-[min(100%,360px)] px-6">
```

Comment:
when doing tailwind classes, the [best practice](https://tailwindcss.com/docs/responsive-design) is to style for mobile first, and then add modifiers like `md:` to things that are specific to larger screens.

also this looks like a min-width class would handle things?